spec/integration/twingly/amqp/utilities_spec.rb

```diff
@@ -60,6 +60,32 @@
           end
         end
       end
+
+      describe "queue_type:" do
+        subject(:queue_type) { created_queue.arguments["x-queue-type"] }
+
+        context "by default" do
```

Comment:
```suggestion
        context "with default value" do
```

(I know that all occurrences in this file are like your version, so maybe we should stick to it in this public repo)

Reply:
Since there are two other occurrences in the same file, I think we can stick with this for now and solve them all in a separate PR later.